terraform/modules/emr/templates/emr/hdfs_setup.sh

```diff
@@ -35,3 +37,8 @@ sudo systemctl start hive-server2
 sudo -H -u hdfs bash -c "hdfs dfs -mkdir /libs"
 sudo -H -u hdfs bash -c "hdfs dfs -put /usr/lib/hbase/hbase-client.jar /libs"
 
+for EACH_GROUP in $${USER_GROUPS[@]}; do
+  echo "Creating group DB in S3 for '$EACH_GROUP'"
+  str=$(echo $EACH_GROUP | sed -e 's/[^a-zA-Z0-9_]/_/g')
+  sudo /bin/hive -e "CREATE DATABASE IF NOT EXISTS $${str}_db LOCATION '$S3_FILE_PATH/$str/$${str}_db'"
```

Comment:
Whats with the double $$? The str wont be env var, so ${str} so won't $${str}_db always resolve to '_db'?

Reply:
Why we using $$?  Looks like the previous version worked fine?

Reply:
It is a template file so tf is looking for `${<value>}` to template into it - the `$$` means that tf leaves it alone and it is able to plan/apply without complaining about missing template vars.